components/alert/alert.css

```diff
@@ -0,0 +1,91 @@
+* {
+    margin: 0;
+    padding: 0;
+}
+/* @media screen and (max-width: 800px) */
+
+.doc-content {
+    margin: auto;
+    width: 90%;
+    min-width: 0;
+    max-width: 800px;
+    flex: 1 1 0%;
+
+}
+.content-title {
+    font-size: 1.5rem;
+    font-family: sans-serif;
+    font-weight: 900;
+    margin-bottom: 1rem;
+}
+.content-subtitle {
+    font-size: 1rem;
+    font-family: sans-serif;
+    font-weight: 400;
+    margin-bottom: 1rem;
+}
+.divider {
+    margin-bottom: 1rem;
+    width: 100%;
+    height: 1px;
+    background-color: #dadadaf3;
+}
+.content-head {
+    font-size: 1.5rem;
+    font-family: sans-serif;
+    font-weight: 700;
+    margin-bottom: 1rem;
+}
+.alert-box {
```

Comment:
`.alert-box` and `.example-div` seem to serve the same purpose. Maybe you can merge the classes and make only one class. Also with `flex-direction: column` you dont need `flex-wrap` (Because you want the alerts to appear one below the other)